modin/core/execution/ray/common/task_wrapper.py

```diff
@@ -17,6 +17,8 @@
 To be used as a piece of building a Ray-based engine.
 """
 
```

Comment:
@anmyachev, do you think this file should be renamed (that should land onto a separate PR)?